### PR TITLE
[v6r20] Docs: rewrite of RMS doc

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES
Please follow the template:
*Docs
CHANGE: Complete RMS doc

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Since you are on this, can you remove file:
docs/source/AdministratorGuide/Configuration/ConfReference/Systems/DataManagement/Agents/TransferAgent/index.rst
and check what it is in
docs/source/AdministratorGuide/Configuration/ConfReference/Systems/DataManagement/

?

**Comment by chaen** [comment]
In accordance to what we agreed in https://github.com/DIRACGrid/DIRAC/pull/3680, and that we are meant to discuss next week, everything under /AdministratorGuide/Configuration/ConfReference/ should go. Do we wait the discussion of next week, or do I just remove the whole folder ?

**Comment by fstagni** [comment]
Wait for it!

File: docs/source/AdministratorGuide/Systems/RequestManagement/concepts.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+When an `Operation` acts on LFNs, `Files` corresponding to the LFNs are associated to the `Operation`.
+
+The list of available `Operations`, as well as the state machines are described in :ref:`rmsObjects`
+
+ReqManager & ReqProxy
+---------------------
+
+The `ReqManager` is the service that receives or distributes `Requests` to be excuted. Every operation is synchronous with the `ReqDB` database.
+
+In the ReqManager is unreachable when a client wants to send a `Request`, the client will automatically failover to a `ReqProxy`. This proxy will accept the `Request`, hold it in a local cache, and will periodically try to send it to the `ReqManager` until it succeeds. This system ensures that no Request is lost.
```
Comment (fstagni) [inline docs/source/AdministratorGuide/Systems/RequestManagement/concepts.rst]: "If the ReqManager... "

File: docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+Remove the replica of a file at a given SE and from the FC
+
+
+Details: :py:mod:`~DIRAC.DataManagementSystem.Agent.RequestOperations.RemoveReplica`
+
+
+ReplicateAndRegister
+====================
+
+This Operation replicate a file to one or several SE. The source does not need to be specified, but can be. This is typically useful in case of failover: if a job tries to upload a file to its final destination and fails, it will upload it somewhere else, and creates a `ReplicateAndRegister` Operation as well as a `RemoveReplica` (from the temporary storage) Operation. The replication can be performed either locally, or delegating it to the FTS system (:ref:`fts3`)
```
Comment (fstagni) [inline docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst]: replicateS

**Comment by chaen** [comment]
https://i.pinimg.com/originals/94/24/20/942420175b984bebc86322490904ce14.jpg

File: RequestManagementSystem/Agent/RequestExecutingAgent.py [hunk 3]
```diff
@@ -158,6 +158,7 @@ def __init__( self, *args, **kwargs ):
     # # create request dict
     self.__requestCache = dict()
 
+    # ?? Probably should be removed
```
[the remark this answers is not in the document]
Comment (chaen) [reply RequestManagementSystem/Agent/RequestExecutingAgent.py]: It is in the ReplicateAndRegister configuration itself, not the RequestExecutingAgent

[the remark this answers is not in the document]
Comment (fstagni) [reply RequestManagementSystem/Agent/RequestExecutingAgent.py]: ... remove it then!

**Comment by fstagni** [comment]
Can this one be merged, or you want to add more to it?

**Comment by chaen** [comment]
Please merge already, if needed I'll add another one

**Pull request merged**